Arduino/McLighting/McLighting.ino

```diff
@@ -692,6 +771,93 @@ void setup() {
     getStatusJSON();
   });
 
+  server.on("/pixels", []() {
```

Comment:
As said, this looks fine to me.

Reply:
One more addition: I like it, that you put all three operations in one endpoint, but I think `pixels` is a bit misleading. Maybe change it to `pixelconf`?

Reply:
We can use anything here, I just went with what came to my thoughts at that moment. I like the idea of using `pixelconf`. We can add the HTTP method later, restricting to only POST when the web interface to set these up is complete. That way users have a way to communicate and set pixel configuration.

Reply:
Changed to `pixelconf` in https://github.com/toblum/McLighting/pull/350/commits/1f0087fec856708075e9fe087d92b4d0dc8001d6